pkg/config/sdk.go

```diff
@@ -81,6 +77,10 @@ func (s *SdkInfo) UnmarshalYAML(data *yaml.Node) (err error) {
 			}
 		}
 	}
+	if !a.Layer.Validate() {
```

Comment:
I think the same thing applies here, this check should be moved inside the annotators length check